Adjust introductory sentences

andymunro opened the pull request:
Closes #39080

Initial suggestions. More to come.

andymunro committed:
Adjust introductory sentences

Closes #39008

Signed-off-by: AndyMunro <amunro@redhat.com>

andymunro commented:
@ahus1 Please review

ahus1 reviewed (changes requested):
See below for some suggestions. 
This touches several guides, and I assume we need approval from the respective teams. So I suppose we need separate ones for operator, securing-apps and server. 

Starting the summary with "Learn how to..." is one way to go, still when is IMHO one of the "weak" verbs to use here. Once we use it in  a lot of places, it starts to sound redundant and doesn't reveal much about the information that is about to follow.

andymunro commented:
@ahus1 I reduced the amount "Learn how to" and addressed your comments.  I suggest we agree on a consistent approch before converting this to multiple MRs for approval by different engineers.

ahus1 reviewed (dismissed):
Thank you for this change, this now looks good to me. I added a missing quote, and extended some of the texts. Please confirm. 

I like the updated titles and shorter summaries.

vmuzikar reviewed (dismissed):
LGTM, thanks @andymunro

andymunro commented:
@ahus1 I realized that the Getting Started summaries were not in a sentence format. I updated them to be consistent.

andymunro commented:
@ahus1 Also, I integrate the changes from your commit into my commit, which was probably not needed. In any case, the only change as a result is to the getting started summaries. If they are okay, we are ready to merge.

ahus1 commented:
@andymunro - yes, squashing the commits wasn't necessary. Waiting a bit longer for Marek to confirm the PR changes, otherwise we'll just proceed.

andymunro committed:
Merge branch 'main' into 39080

ahus1 reviewed (approved):
Thank you for this change. Assuming lazy consensus.
